src/main/drivers/sdcard_spi.c

```diff
@@ -152,15 +193,25 @@ static bool sdcard_waitForIdle(int maxBytesToWait)
  */
 static uint8_t sdcard_waitForNonIdleByte(int maxDelay)
 {
-    for (int i = 0; i < maxDelay + 1; i++) { // + 1 so we can wait for maxDelay '0xFF' bytes before reading a response byte afterwards
-        uint8_t response = spiBusTransferByte(&sdcard.busdev, 0xFF);
+    uint8_t idleByte;
 
-        if (response != 0xFF) {
-            return response;
-        }
-    }
+    // Note that this does not release the CS at the end of the transaction
+    busSegment_t segments[] = {
```

Comment:
Should this be `static const`?

Reply:
It doesn't need to be static as the `spiWait()` call on exit blocks and so doesn't allow this to go out of scope during the DMA. It could be const, but then the call to `spiSequence()` would barf saying we'd discarded the qualifier, so I don't think it should be.